portfolio/src/main/webapp/index.html

```diff
@@ -134,7 +136,15 @@ <h2>Misc</h2>
     </div>
     <!--Comments-->
     <div class="comment_background">
-      <div id="comments_sec"></div>
+      <div id="comments_sec">
+        <h2>Leave a Comment!</h2>
+        <form action="/comments" method="POST">
+          <textarea name="comments-input"class="form-control" rows="6";></textarea>
```

Comment:
nit: add a space between `name` and `class` and remove the semicolon after `rows`